src/Model/Config/ProjectConfig.php

```diff
@@ -42,6 +43,10 @@ class ProjectConfig
     public const KEY_RECIPES = 'recipes';
     public const KEY_TASKS = 'tasks';
     public const KEY_UPSTREAMS = 'upstreams';
+    public const KEY_COMPOSER_PATH = 'composer_path';
```

Comment:
Leaving these new keys here for now too, but I think these will ultimately be moved into a different file. Some sort of config deriver class should own this instead of the project config. Or if these should stay in config, we should put them into a `~/.waffle.yml` file instead.

The `drush_major_version` is already available in the `DrushCommandRunner` class. We should just have to expose that.

Reply:
Followup Issue: https://github.com/daceej/waffle/issues/34